ru.linux_gaming.PortProton.yml

```diff
@@ -0,0 +1,333 @@
+app-id: ru.linux_gaming.PortProton
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '23.08'
+base: org.winehq.Wine
+base-version: stable-23.08
+command: portproton
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
why do you need both wayland and x11?

Reply:
If I do 

```
  --socket=wayland
  --socket=fallback-x11
```
The application will not start in Wayland

Reply:
then why do you still have wayland?

Reply:
> then why do you still have wayland?

As it is now, PortProton runs smoothly in both X11 and Wayland, if I do what x11-fallback recommends, nothing will work.

Reply:
that mean it doesn't support wayland and still require X11. (Xwayland isn't wayland, it's X11, just that instead of calling the graphics driver it calls Wayland)

Reply:
> that mean it doesn't support wayland and still require X11. (Xwayland isn't wayland, it's X11, just that instead of calling the graphics driver it calls Wayland)

So I have to remove --socket=wayland

Reply:
you don't need it you don't put it.

Reply:
> you don't need it you don't put it.

Will the application in Xwayland work without it ?

Reply:
I checked, I don't need it, I'll remove it.

Reply:
>  (Xwayland isn't wayland, it's X11, just that instead of calling the graphics driver it calls Wayland)

what's unclear?

Reply:
> > (Xwayland isn't wayland, it's X11, just that instead of calling the graphics driver it calls Wayland)
> 
> what's unclear?

I checked and already deleted it